File: mycollab-web-ondemand/src/main/scala/com/mycollab/ondemand/module/file/event/CloudDriveOAuthCallbackEvent.scala
package com.mycollab.ondemand.module.file.event

import com.mycollab.events.ApplicationEvent
import com.mycollab.module.file.CloudDriveInfo

/**
  *
  * @author MyCollab Ltd.
  * @since 1.0
  *
  */
object CloudDriveOAuthCallbackEvent {
  
  @SerialVersionUID(1L)
  class ReceiveCloudDriveInfo(source: AnyRef, data: CloudDriveInfo) extends ApplicationEvent(source, data) {}
  
}

File: mycollab-web-ondemand/src/main/java/com/mycollab/ondemand/module/file/event/CloudDriveOAuthCallbackEvent.kt
package com.mycollab.ondemand.module.file.event

import com.mycollab.vaadin.event.ApplicationEvent
import com.mycollab.module.file.CloudDriveInfo

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class CloudDriveOAuthCallbackEvent {
    class ReceiveCloudDriveInfo(source: Any, val data: CloudDriveInfo) : ApplicationEvent(source)
}